spring-cloud-contract-wiremock/src/main/java/org/springframework/cloud/contract/wiremock/restdocs/WireMockRestDocsConfiguration.java

```diff
@@ -38,9 +43,42 @@
 public class WireMockRestDocsConfiguration
 		implements RestDocsMockMvcConfigurationCustomizer {
 
+	private final Environment environment;
+
+	public WireMockRestDocsConfiguration(Environment environment) {
+		this.environment = environment;
+	}
+
 	@Override
 	public void customize(MockMvcRestDocumentationConfigurer configurer) {
+		if (this.environment.getProperty("wiremock.placeholders.enabled", "true")
+				.equals("true")) {
+			configurer.operationPreprocessors()
+					.withResponseDefaults(new ForwardHeaderPreprocessor());
+		}
 		configurer.snippets().withAdditionalDefaults(new WireMockSnippet());
 	}
 
+	static class ForwardHeaderPreprocessor implements OperationPreprocessor {
+
+		private final OperationResponseFactory responseFactory = new OperationResponseFactory();
+
+		@Override
+		public OperationRequest preprocess(OperationRequest request) {
+			return request;
+		}
+
+		@Override
+		public OperationResponse preprocess(OperationResponse response) {
+			String content = response.getContentAsString();
+			if (content.contains("localhost:8080")) {
```

Comment:
Will it always be `:8080` ?

Reply:
That's the default for MockMvc, and users almost never change it - I expect we could make it configurable if anyone asks. Actually I didn't try the reactive version, but I expect it's the same. Maybe we need a test.